findphotodates.py

```
@@ -0,0 +1,315 @@
+            lines = result.stdout.strip().split("\n")
+            
+            # Process date - try each possible date field
+            date = None
+            for i, line in enumerate(lines):
+                if i < len(date_tags) and line and not line.startswith("GPS"):
+                    try:
+                        # Basic validation that this looks like a date
+                        if re.match(r'\d{4}:\d{2}:\d{2} \d{2}:\d{2}:\d{2}', line):
+                            date = line
+                            break
+                    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)